### DISCUSSION TITLE
Fan Control Won't Start With Windows Since Moving the Program to a Different Drive?

### DISCUSSION THREAD

**Discussion by shoegod49** [Rem0o/FanControl.Releases#2126] [Q&A]
I had to move Fan Control to a different drive because the original drive took a crap. I did this by making sure Fan Control was completely exited, checking task manager to make sure, and then deleting the entire Fan Control folder except for keeping the config file. I then installed Fan Control on the drive it is now on but now Fan Control will not open/run when i turn on my PC. I have made sure to enable "Run at Windows Startup" within the settings, I have added Fan Control to my startup Folder via Shell:startup, I have added Fan Control to Task Scheduler scheduled to startup at boot up. I've basically done everything I can think of to get Fan Control to automatically run at startup but so far nothing has worked.

**Comment by Rem0o** [comment]
Go into Task scheduler, take a look at the FanControl task, make sure it points to the right location.   Then what kind of delay do you got set up? If you put a number too small, it will fail to open most of the time.

**Reply by shoegod49** [reply]
It started working after I changed the delay to 3 seconds, but now just today it stopped working. Also last week the updater was flagged as a trojan "severe risk" by Windows Defender. I restored the updater file and whitelisted it but as mentioned just today Fan Control stopped working. Even going into the Fan Control Folder and right clicking and running as admin does nothing, it just won't load. I'm going to delete the Fan Control Folder, keeping my user config file and reinstall it and see if that works.
